lib/Raven/SerializerInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace Raven;
+
+interface SerializerInterface
+{
+    /**
+     * Serialize an object (recursively) into something safe for data
+     * sanitization and encoding.
+     *
+     * @param mixed $value
+     * @param int   $max_depth
+     * @param int   $_depth
+     * @return string|bool|double|int|null|object|array
+     */
+    public function serialize($value, $max_depth = 3, $_depth = 0);
```

Comment:
>Okay. Which of us will make a pull request that will make all methods protected?

This is off-topic, but I don't think there is a need _now_ to open a PR to change all methods from public to protected because we will probably do it with the future PRs regarding other changes. Mine was just a statement,related to public methods that were strictly connected with the serializer (`Client::serialize` for example): only those should be changed in this PR